packages/cli/src/util/client.ts

```diff
@@ -1,7 +1,11 @@
 import { MedplumClient, MedplumClientOptions } from '@medplum/core';
 import { FileSystemStorage } from '../storage';
+import { Profile } from '../utils';
 
-export async function createMedplumClient(options: MedplumClientOptions): Promise<MedplumClient> {
+export async function createMedplumClient(
+  options: MedplumClientOptions,
+  profileObject?: Profile
```

Comment:
We should not use `Object` as a variable name suffix.  Let's just call this `profile?: Profile`